src/playground-scheduler-functions.ts

```diff
@@ -0,0 +1,93 @@
+import schedulerHelper, { HttpMethod } from './utilities/scheduler-helper';
```

Comment:
Should there be a playground directory, rather than just under src?

Reply:
Not a bad idea, I just moved this out for now
I was thinking we would either restructure or delete the file
it might make sense to put it in a subdirectory and then exclude it from the build?